lib/dai-plugin-mcd/test/CdpTypeService.spec.js

```diff
@@ -12,26 +9,23 @@ let maker, service;
 // these CDP types should be available to the Maker instance because
 // of the configuration passed into it (see test/helpers.js)
 const scenarios = [['ETH-A', ETH], ['ETH-B', ETH], ['COL1-A', COL1]];
-const systemValues = {
-  'ETH-A': {
-    expectedCeiling: 1000,
-    expectedRatio: 1.5,
-    expectedFee: '2.5'
-  },
-  'ETH-B': {
-    expectedCeiling: 500,
-    expectedRatio: 1.8,
-    expectedFee: '1.5'
-  },
-  'COL1-A': {
-    expectedCeiling: 800,
-    expectedRatio: 1.8,
-    expectedFee: '2.0'
-  }
-};
+let systemValues = {};
+
+function setSystemValues() {
```

Comment:
sorry if this seems nitpicky, but what's the advantage of doing it this way rather than just declaring the configuration object directly?

Reply:
I was just trying to avoid defining it literally since it took up so much space, and most of it was repeating the same three keys three times.

Reply:
ah, i see. if i were trying to save space in this case i would probably just use arrays the whole way through, since this is just a test file, so we can be a little more relaxed about structure:
```js
const scenarios = [
  // last three values are expected debt ceiling, liq ratio, and stability fee
  ['ETH-A', ETH, 1000, 1.5, 2.5],
  // etc.
];

// then in tests:
const expectedCeiling = scenarios[ilk][2];
```